docs/ocean/design_docs/prognostic-sediment-transport.rst

```diff
@@ -0,0 +1,519 @@
+:math:`\psi` is the angle between current direction and wave travelling direction, :math:`\theta_w` and :math:`\theta_m`
+are wave-induced Shields parameter and the mean Shields paramter, respectively (i.e. computed by :math:`\tau_w` and
+:math:`\tau_m`, respectively, using the Shields parameter equation).
+:math:`\tau_c` and :math:`\tau_w` are bottom shear stress from the currents and waves, respectively.
+Notice that :math:`\Phi_x = \Phi_y =0` if :math:`\theta_m \leq \theta_{cr}`.
+
+
+**Engelund-Hansen**
+
+One form of the Engelund-Hansen formulation is
+
+.. math::
+
+      q_b = \frac{0.05\overline{U}^5\rho_s} {\sqrt{g}C^3(s-1)^2D_{50}}
+
+in which :math:`\overline{U}` is the magnitude of the depth-averaged velocity (:math:`m/s`), :math:`C`
+is the Chezy coefficient (:math:`m^{1/2} s^{-1}`).
+
+
+Algorithm Design: Suspended sediment transport
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2021/01/17
+
+Contributors: Zhendong Cao
+
+**Empirical approach**
+
+When sediment is in suspension, the settling towards the bed is counterbalanced by the upward diffusion of the sediment due to
+the turbulence eddies. Thus, SSC profile :math:`C(z)` can be obtained by solving the following equations
+
+.. math::
+
+      C(z) w_s + \epsilon_s \frac{dC(z)}{dz} = 0
+
+in which :math:`C(z)` is the suspended sediment concentration (:math:`kg m^{-3}`), :math:`w_s` is the sediment settling velocity
+(:math:`ms^{-1}`), :math:`\epsilon_s` is the sediment diffusion coefficient (:math:`m^2s^{-1}`), and :math:`z` is the vertical
+coordinate (:math:`m`) with positive upward. Assuming the eddy diffusivity varying parabolically with height, the solution of
+:math:`C(z)` can be written as
+
+.. math::
+
+      C(z) = C_{ref}\left[ \frac{z_{ref}}{z} \frac{H-z}{H-z_{ref}} \right]^b
+
+in which :math:`C_{ref}` is the near-bottom reference SSC (:math:`kg m^{-3}`),
+:math:`z_{ref}` is the reference height (:math:`m`) of :math:`C_{ref}`,
+:math:`H` is the total water depth (:math:`m`),
+and :math:`b` is the suspension parameter called Rouse number, which is calculated by
+
+.. math::
+
+      b = \frac{w_s}{\kappa u_*}
+
+where :math:`\kappa` is the von Karman constant (0.4) and :math:`u_*` is the shear friction velocity (:math:`ms^{-1}`).
+The shear stress :math:`\tau = C_d \overline{U}^2={u_*}^2`, so :math:`u_*={C_d}^{0.5} \overline{U}`.
+
+We provde two options to compute :math:`C_{ref}` and :math:`z_{ref}`:
+
+- *Zyserman and Fredsoe, 1994*
+
+.. math::
+
+      C_{ref} = \frac{0.331(\theta - 0.045)^{1.75}}{1+0.72(\theta-0.045)^{1.75}}
+
+and :math:`z_{ref}=2D_{50}`. Again, :math:`\theta` is the Shields parameter and
+:math:`D_{50}` is the sediment median grain diameter.
+
+- *Goldstein et al., 2014*
+
+.. math::
+
+      C_{ref} = \left[ \frac{0.328U_b}{0.0688+1000D_{50}}\right]^2
+
+at height :math:`z_{ref}=0.01 m`, where :math:`U_b` is the magnitude of the bottom velocity (:math:`ms^{-1}`)
+
```

Comment:
Based on this link: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/lists.html#code-images-and-other-content-inside-lists

```suggestion
- *Zyserman and Fredsoe, 1994*

  .. math::

      C_{ref} = \frac{0.331(\theta - 0.045)^{1.75}}{1+0.72(\theta-0.045)^{1.75}}

  and :math:`z_{ref}=2D_{50}`. Again, :math:`\theta` is the Shields parameter and
:math:`D_{50}` is the sediment median grain diameter.

- *Goldstein et al., 2014*

  .. math::

      C_{ref} = \left[ \frac{0.328U_b}{0.0688+1000D_{50}}\right]^2

  at height :math:`z_{ref}=0.01 m`, where :math:`U_b` is the magnitude of the bottom velocity (:math:`ms^{-1}`)

```

Reply:
That is, everything but the "bullet" lines need to be indented 2 spaces.

Reply:
Got it.